### DISCUSSION TITLE
Override http status code for denied access

### DISCUSSION THREAD

**Discussion by R-Nabil** [NginxProxyManager/nginx-proxy-manager#2216] [General]
Hi,

Currently, if someone not on the access list of a subdomain tries to reach he will be served with a 403 forbidden error.
I'd like to override that error to something different/custom. Is there a way that can be done ?

Thanks,

**Comment by reallango** [comment]
I am also trying to figure something out for this. I have some internal only sites setup. I would like to have them forward to my 404 page for my main site if accessed from the outside.

**Comment by reallango** [comment]
Ok after further testing and digging I found a solution that works nicely for me. In the advanced section for my internal only proxy hosts I put the following code and it now redirects to my standard 404 page. This way you can't tell that there is a subdomain available to access with this name.

error_page 403      https://example.com/404/;

found at the following site. https://www.cyberciti.biz/faq/howto-nginx-customizing-404-403-error-page/
